jest.config.ts

```diff
@@ -0,0 +1,58 @@
+import type { Config } from '@jest/types';
```

Comment:
```suggestion
import type { Config } from 'jest';
```

https://jestjs.io/docs/configuration